### PR TITLE
[ICDS] updates for GA

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz,

I updated Google Analytics for ICDS, now we should in an easy way to see the events for specific URL in GA.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

**Comment by lwyszomi** [comment]
The solution implemented according to the google specification: https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications

File: custom/icds_reports/static/js/icds_app.js [hunk 1]
```diff
@@ -122,181 +124,194 @@ window.angular.module('icdsApp', ['ngRoute', 'ui.select', 'ngSanitize', 'datamap
...
-                template : "submitted",
+                template: "submitted",
             })
             .when("/system_usage_tabular", {
-                template : "system_usage_tabular",
+                template: "system_usage_tabular",
             })
-            .when("/underweight_children", {
-                redirectTo : "/underweight_children/map",
+            .when("/comp_feeding", {
```
Comment (calellowitz) [inline custom/icds_reports/static/js/icds_app.js]: Are these changes related to the GA work?

**Review by calellowitz — COMMENTED** [review]
Most of it looks good. one brief questions

File: custom/icds_reports/static/js/icds_app.js (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/static/js/icds_app.js]: @calellowitz yes these changes are related. This looks strange because I sorted the urls, but I added only group for the reports urls. For ex: we had ```/underweight_children``` but after my change we will have ```/maternal_and_child/underweight_children```

**Review by calellowitz — APPROVED** [review]

**Comment by lwyszomi** [comment]
@calellowitz (bump)
